rst_files/lucas_model.rst

```diff
@@ -414,12 +414,109 @@ Let's try this when :math:`\ln y_{t+1} = \alpha \ln y_t + \sigma \epsilon_{t+1}`
 
 Utility will take the isoelastic form :math:`u(c) = c^{1-\gamma}/(1-\gamma)`, where :math:`\gamma > 0` is the coefficient of relative risk aversion
 
-Some code to implement the iterative computational procedure can be found in
-`lucastree.jl <https://github.com/QuantEcon/QuantEcon.lectures.code/blob/master/lucas_model/lucastree.jl>`__
+Some code to implement the iterative computational procedure can be found below:
 
-We repeat it here for convenience
+.. code-block:: julia
+
+    #=
+
+    @authors : Spencer Lyon <spencer.lyon@nyu.edu>, John Stachurski
+
+    =#
+
+    using QuantEcon, Distributions
+
+    """
+    The Lucas asset pricing model --- parameters and grid data
+    """
+    struct LucasTree{TF<:AbstractFloat}
```

Comment:
This def needs be re-written afterwards.

Reply:
For sure. Lots of boilerplate that we can eliminate.